README.md

```diff
@@ -91,16 +91,34 @@ Run `yarn clean-all` if the `yarn start` command fails due to a change in the co
+every week can be defined in the `config/grain.json` file. 
+
+There are three different policies that can be used to control
 how the grain gets distributed: 
+
 - `immediatePerWeek` splits the grain evenly based on everyone's Cred in the last week only.
-- `balancedPerWeek` distributes the grain consistently based on total lifetime cred scores. i.e. it balances
-the distribution of grain with the distribution of total historical cred.
+- `balancedPerWeek` distributes the grain consistently based on total lifetime Cred scores. i.e. it balances
+the distribution of grain with the distribution of total historical Cred.
+- `recentPerWeek` distributes the grain based on an exponential decay, such that more recent contributions receive more grain than older contributions.
+
+You can choose to distribute grain using any one of these policies, or a weighted combination of all three. Simply specify the amount of grain to be distributed by each policy. For instance, in the example below, we tell SourceCred to distribute 1,000 grain every week, with 25% (250 grain) distributed according to `immediatePerWeek`, 25% (250 grain) distributed according to `balancedPerWeek`, and 75% (750 grain) distributed according to `recentPerWeek`. 
+
+```
+{
+ "immediatePerWeek": 250,
+ "balancedPerWeek": 250,
+ "recentPerWeek": 750,
+ "recentWeeklyDecayRate": 0.5,
+ "maxSimultaneousDistributions": 100
+}
+```
+
+The `balancedPerWeek` policy allows SourceCred to reward people retro-actively. For example, If someone has been historically "overpaid"
+with grain relative to their Cred scores, that person will receive less grain in the balanced distribution while people
+who have been "underpaid" relative to their Cred will receive more grain.
 
-The balanced policy allows SourceCred to reward people retro-actively. e.g. If someone has been historically "overpaid"
-with grain relative to their cred scores, that people will receive less grain in the balanced distribution while people
-who have been "underpaid" relative to their cred will receive more grain.
+The `recentPerWeek` policy weights contributions with an exponential decay. The `recentWeeklyDecayRate` parameter determines the exponential rate of decay. The weighting for each week is equal to (1 - `recentWeeklyDecayRate`) raised to the power of the week number. For example, if `recentWeeklyDecayRate` is set to 1, this policy is equivalent to the `immediatePerWeek` policy; 100% of grain is distributed based on Cred generated in the last week and Cred created in prior weeks is "discounted" to 0%. If `recentWeeklyDecayRate` is set to 0.5 (i.e. 50% discount), as in the above example, the policy will count 100% of the Cred generated in the last week, 50% of the Cred generated in the week before, 25% from the week before that, 12.5% from the week before that, and so on. In other words, `recentPerWeek` is a lot more flexible in how it rewards the most recent contributions without making it a hard 1-week cut-off.
```

Comment:
This is necessary information, but I tend to prefer README's as short as possible, in this case I think of it as the minimum information needed to get an instance running.  I suggest instead linking to our docs on the details of these policies with a line like:

### FAQ:
- [What does each policy do?]()
- [How should I choose the right policy for me?]()
- [Can I change the policies in the future?]()

Reply:
+1 to this.

Reply:
Agreed. @elihanover, when your PR adding this info to the Grain page of the docs is merged, I'll implement your above suggested change. Just left a quick [review on your PR](https://github.com/sourcecred/docs/pull/250#pullrequestreview-564682249).